docs/content/examples/tutorials/mailserver-behind-proxy.md

```diff
@@ -72,14 +72,15 @@ Feel free to add your configuration if you achieved the same goal using differen
           - "traefik.tcp.services.smtp.loadbalancer.server.port=25"
           - "traefik.tcp.services.smtp.loadbalancer.proxyProtocol.version=1"
           - "traefik.tcp.routers.smtp-ssl.rule=HostSNI(`*`)"
-          - "traefik.tcp.routers.smtp-ssl.tls=false"
           - "traefik.tcp.routers.smtp-ssl.entrypoints=smtp-ssl"
+          - "traefik.tcp.routers.smtp-ssl.tls.passthrough=true"
```

Comment:
👍 

---

Reference:
- https://doc.traefik.io/traefik/routing/routers/#passthrough
- https://traefik.io/blog/traefik-2-tls-101-23b4fbee81f1/#what-about-passthrough

Doesn't require `tls=false`, but should hand the traffic over to DMS on port 465 encrypted, and DMS will handle the connection (_and presumably certs_).

Related:
- [`HostSNI` must either be an explicit value with no wildcard, or only `*`](https://stackoverflow.com/a/60858441).
- Ports that use StartTLS instead should not proxy through Traefik for TLS/certs, they need to establish a plain-text connection to DMS ports to upgrade to TLS explicitly, so `.tls=false` is appropriate for those TCP router port configs AFAIK as Traefik does not support StartTLS:
  - https://github.com/docker-mailserver/docker-mailserver/issues/3563#issuecomment-1744653719
  - https://github.com/docker-mailserver/docker-mailserver/issues/3563#issuecomment-1745728038